defaults/main.yml

```diff
@@ -4,3 +4,6 @@ server:
 php:
     ppa: "php5-5.6"
     packages: ["php5-cli", "php5-intl", "php5-mcrypt"]
+
+php_version: "{% if php.ppa == 'php-7.0' %}php7.0{% else %}php5{% endif %}"
+php_config_prefix: "{% if php.ppa == 'php-7.0' %}/etc/php/7.0{% else %}/etc/php5{% endif %}"
```

Comment:
@popstas is this differantionation a requirement? If the installed version is only one all the time I would probably just use the default destination for the configuration files. Also I'm not fully sure about using conditionals in the vars files. Do you think we can move that into the task file as runtime gathered fact?